proto/proto/store/ntx_builder.proto

```diff
@@ -60,6 +63,28 @@ message UnconsumedNetworkNotesRequest {
     uint64 page_size = 2;
 }
 
+// Returns a list of unconsumed network notes for an account up to a specified block number using
+// pagination.
+message UnconsumedNetworkNotesForAccountRequest {
+    // This should be null on the first call, and set to the response
+    // token until the response token is null, at which point all data
+    // has been fetched.
+    //
+    // Note that this token is only valid if used with the same parameters.
+     optional uint64 page_token = 1;
+
+     // Number of notes to retrieve per page.
+     uint64 page_size = 2;
+
+     // The network account ID prefix to filter notes by.
+     uint32 network_account_id_prefix = 3;
+
+     // The maximum (inclusive) block number to scope the query by.
+     //
+     // Notes that are consumed after this block are excluded from the result.
+     fixed32 latest_block_num = 4;
```

Comment:
I still don't think the comment here is correct. We are not excluding the notes consumed after this block number in the result right? Instead, we are including them - or am I missing something. Shouldn't the comment read:

The result includes notes which: (1) created before this block and (2) haven't been consumed by this block.

Reply:
Yea sorry you are right. I must have been thinking consumption is ignored after that block. I do find this query and endpoint quite confusing still. Will update soon.

Reply:
LMK how you think that reads now. Probably still room for improvement.